src/locales/messages.pot

```diff
@@ -3422,6 +3428,9 @@ msgstr ""
 msgid "When distributing, payouts to Ethereum addresses incur a 2.5% JBX membership fee. Payouts to other Juicebox projects don't incur fees. Your project will receive (the <0>JuiceboxDAO</0> token) in return at the current issuance rate."
 msgstr ""
 
+msgid "When enabled, all transfers will be paused."
```

Comment:
transfers to project tokens will be paused. This does not apply to ERC-20 tokens if issued.